go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/AdamKorcz/go-118-fuzz-build v0.0.0-20230306123547-8075edf89bb0
 	github.com/Microsoft/go-winio v0.6.1
 	github.com/Microsoft/hcsshim v0.12.0-rc.0
+	github.com/Microsoft/hcsshim/test v0.0.0-20210227013316-43a75bb4edd3
```

Comment:
Why do we have import of this module. Why didnt we have this earlier? Because earlier also, it was being used in windows tests.

Reply:
```
Commit 1: Temporarily remove the integration/client submodule. I could not figure out a clean way to move this module to use the new v2 module name since go mod will always fail lookup since there is no tag or commit with the new module name. I'm not sure how this was ever supposed to work, but replace doesn't seem to do the trick for me.
```

I believe this was the only reason this submodule was created. We will add it back if still necessary, something was complaining about the checked in `rsrc_amd64.syso`, but no issues with CI now.